.vscode/extensions.json

```
@@ -1,8 +1,9 @@
 {
-    "recommendations": [
-      "ms-python.python",
-      "charliermarsh.ruff",
-      "dbaeumer.vscode-eslint",
-      "tamasfe.even-better-toml",
-    ]
-  }
\ No newline at end of file
+  "recommendations": [
+    "ms-python.python",
+    "charliermarsh.ruff",
+    "dbaeumer.vscode-eslint",
+    "tamasfe.even-better-toml",
+    "batisteo.vscode-django"
+  ]
```

The indent changed from two spaces to four, I'd suggest leaving the indentation as it was:

```suggestion
    "recommendations": [
      "ms-python.python",
      "charliermarsh.ruff",
      "dbaeumer.vscode-eslint",
      "tamasfe.even-better-toml",
      "batisteo.vscode-django"
    ]
```